### PR TITLE
add logic to change panels between list and profile

### PR DESCRIPTION
**Pull request by Ben-Hanan**
Work in progress. Basic functionality of switching panels is present but the individual components need to be made and styled. I've provided some skeleton so just follow the implementation. Feedback on the implementation is welcome. I've not added javadocs for some of the files as they might change.

### CODE REVIEW

File: src/main/java/com/eva/logic/commands/ListCommand.java [hunk 1]
```diff
@@ -12,13 +14,29 @@
...
-        model.updateFilteredPersonList(PREDICATE_SHOW_ALL_PERSONS);
-        return new CommandResult(MESSAGE_SUCCESS);
+        if (isStaffList) {
+            model.setPanelState(PanelState.STAFF_LIST);
+            model.updateFilteredStaffList(PREDICATE_SHOW_ALL_STAFFS);
+        } else {
+            model.setPanelState(PanelState.APPLICANT_LIST);
+            model.updateFilteredApplicantList(PREDICATE_SHOW_ALL_APPLICANTS);
+        }
+        return new CommandResult(MESSAGE_SUCCESS, false, false, true);
```
Comment (TheSpaceCuber) [inline src/main/java/com/eva/logic/commands/ListCommand.java:45]: Perhaps you can give a boolean name to the false & true for better readability?

**Review by TheSpaceCuber — COMMENTED** [review]
LGTM

**Comment by IsaacTin** [comment]
lgtm

File: src/main/java/com/eva/logic/commands/ListCommand.java (hunk 1 above)
Comment (Ben-Hanan) [reply src/main/java/com/eva/logic/commands/ListCommand.java:45]: I don't think it's needed cause the attribute in command result would explain it!

**Comment by Hou-Rui** [comment]
LGTM.

**Comment by IsaacTin** [comment]
lgtm

**Pull request merged**